src/main/java/hudson/plugins/clover/CloverPublisher.java

```diff
@@ -272,6 +273,10 @@ private boolean copyHtmlReport(FilePath coverageReport, FilePath buildTarget, Ta
             return false;
         }
         final FilePath htmlDirPath = htmlIndexHtmlPath.getParent();
+        if (htmlDirPath == null) {
```

Comment:
Is this possible to have a directory without a parent? Only for `/`, I guess? But is it even possible to have such Jenkins configuration?

Reply:
I doubt that it is possible to have a directory without a parent.  I didn't want to wrestle with the spotbugs warning that appears if `htmlDirPath` is not checked for null.  It seemed easier to check for null than to insert a spotbugs suppression.

Reply:
OK